apigatewaysetup.py

```diff
@@ -475,26 +475,26 @@ def create_put_method(self, api_gateway, rest_api_id, rest_api_root_id):
                 }
             )
 
-            # Put a 200 integration response in the POST method
-            api_gateway.put_integration_response(
-                restApiId=rest_api_id,
-                resourceId=rest_api_root_id,
-                httpMethod="PUT",
-                statusCode="200",
-                responseParameters={
-                    "method.response.header.Set-Cookie": (
-                        "integration.response.body.Set-Cookie"),
-                    "method.response.header.Access-Control-Allow-Credentials": (
-                        "\'true\'"),
-                    "method.response.header.Access-Control-Allow-Origin": (
-                        "\'https://s3.amazonaws.com\'")
-                },
-                responseTemplates={
-                    "application/json": "$input.body"
-                }
-            )
-
-            # Put a 400 method response in the POST method
+            # # Put a 200 integration response in the PUT method
+            # api_gateway.put_integration_response(
+            #     restApiId=rest_api_id,
+            #     resourceId=rest_api_root_id,
+            #     httpMethod="PUT",
+            #     statusCode="200",
+            #     responseParameters={
+            #         "method.response.header.Set-Cookie": (
+            #             "integration.response.body.Set-Cookie"),
```

Comment:
this is temporary there 
![image](https://user-images.githubusercontent.com/16250861/28555660-60301460-7155-11e7-9805-4dfd64707af3.png)